src/components/forms/DraftFileUpload.tsx

```diff
@@ -10,91 +10,93 @@ export type DraftFileUploadProps = {
+                to={`#`}
+            />
+            <ConfirmationDialog
+                title='Delete Attachment'
+                openState={open}
+                close={handleClose}
+                action={() => {
+                    setOpen(false);
+                    toggleSpinnerRef(true);
+                    handleDeleteFile().then(() => {
+                        toggleSpinnerRef(false);
+                        if (inputRef.current) {
+                            inputRef.current.value = '';
+                        }
+                    });
+                }}
+                description={`Are you sure you want to delete the file '${defaultValue.fileName}'?`}
+            />
+        </> : <div/>;
 
-  const handleUploadButtonClick = () => {
-    inputRef.current?.click();
-  };
+    const handleUploadButtonClick = () => {
+        inputRef.current?.click();
+    };
 
-  const toggleSpinnerRef = (visible: boolean) => {
-    if (spinnerRef?.current) {
-      if (visible) {
-        spinnerRef.current.style.display = 'inline';
-      } else {
-        spinnerRef.current.style.display = 'none';
-      }
-    }
-  };
+    const toggleSpinnerRef = (visible: boolean) => {
+        if (spinnerRef?.current) {
+            if (visible) {
+                spinnerRef.current.style.display = 'inline';
+            } else {
+                spinnerRef.current.style.display = 'none';
+            }
+        }
+    };
 
-  return <div>
-    <FormFieldTitle
-      formId={id}
-      title={title}
-      description={description}
-      required={required}
-    /><input type={'file'} ref={inputRef} style={{display: 'none'}} onChange={(event) => {
-      toggleSpinnerRef(true);
-      handleAddFile(event, id).then(() => {
-        toggleSpinnerRef(false);
-      });
+    return <div>
+        <FormFieldTitle
+            formId={id}
+            title={title}
+            description={description}
+            required={required}
+        /><input type={'file'} ref={inputRef} style={{display: 'none'}} onChange={(event) => {
+        toggleSpinnerRef(true);
+        handleAddFile(event, id).then(() => {
+            toggleSpinnerRef(false);
+        });
     }}/>
-    <div style={{display:'inline', margin:'auto'}}>
-      <button disabled={defaultValue != null} onClick={handleUploadButtonClick}>Upload a file</button>
-      {defaultValue && <span style={{marginLeft:'15px'}}>{defaultValue.fileName} {deleteButton}</span>}
-      <div ref={spinnerRef} style={{display: 'none', textAlign: 'center', height: '44', width: '180'}}>
-        <img src={loadingIndicator} alt={'Loading'}/>
-      </div>
-    </div>
-  </div>;
+        <div style={{display: 'inline', margin: 'auto'}}>
+            <button className={'button-complex-submission'} disabled={defaultValue != null}
+                    onClick={handleUploadButtonClick}>Upload a file<span
+                className={'button-icon button-icon-file-upload'} style={{marginLeft: '8px'}}/></button>
```

Comment:
Button style for file upload.

![image](https://github.com/user-attachments/assets/fde8d475-e56f-4b97-9dba-c2f93c54b296)